### ISSUE TITLE
Cannot change the number of generated scrambles for 3x3 multiple blindfolded when a suggested number is present

### ISSUE THREAD

**Issue by viroulep** [thewca/tnoodle#896]
## Actual (problem) behavior

When generating scrambles for a competition with multiple blindfolded, I can't change the suggested number of scrambles generated per attempt.

## Expected behavior

I can change the number of scrambles generated per attempt.
The number seems based on the personal best of the first person on the psychsheet, but I'd definitely expect to be able to generate a few more as they may attempt to break their best result.

## Steps To Reproduce

I think WST members can manage any competition, so looking at http://localhost:2014/scramble?competitionId=PourGesteJaiGateau2024 should just work; otherwise one would have to use a competition with multi and with competitors registered for the event and a personal best.

## Desktop
 - OS: Debian bookworm
 - Browser **including version**: Firefox 121.0, Google Chrome 120.0.6099.199
 - TNoodle Version: 1.2.1
 - Java version: 
 ```
 $ java -version
openjdk version "17.0.8" 2023-07-18
OpenJDK Runtime Environment (build 17.0.8+7-Debian-1deb12u1)
OpenJDK 64-Bit Server VM (build 17.0.8+7-Debian-1deb12u1, mixed mode, sharing)
```

## Additional context

I did not investigate, but it looks like a react state not being updated upon the user input.

## Workaround

I generated the multi scrambles on their own manually.

**Comment by gregorbg** [comment]
Yup, that was a React state conflicting with the WCIF extension.
Fixed on `master`, just needs to be packed into an actual release

**Comment by viroulep** [comment]
Awesome thanks!
